third_party/ti_simplelink_sdk/BUILD.gn

```diff
@@ -39,19 +39,22 @@ config("ti_simplelink_mbedtls_config") {
 
   include_dirs = [
     "${chip_root}/src/platform/cc13x2_26x2",
+    "${chip_root}/src/platform/cc13x2_26x2/crypto",
     "${ti_simplelink_sdk_root}/source",
-    "${ti_simplelink_sdk_root}/source/ti/thread/cc13x2_26x2/libmbedcrypto/config",
   ]
 }
 
 mbedtls_target("mbedtls") {
+  # Hardware acceleration
   sources = [
-    # XXX: fill in hardware accelerated versions, how to link sysconfig dependent code?
+    "${chip_root}/src/platform/cc13x2_26x2/crypto/aes_alt.c",
+    "${chip_root}/src/platform/cc13x2_26x2/crypto/ecjpake_alt.c",
```

Comment:
EC-JPAKE is not used by Matter. Why is this being included in here? It's allowed (since it's in platform), but I'm curious why it enters the picture.

Reply:
This is a copy of the mbedtls acceleration files we have for Thread. Thread's commissioning is not used by Matter, so I can remove the extra files if you wish.